src/main/javascript/modules/area/sign-posts.ts

```diff
@@ -0,0 +1,730 @@
+/**
+ * Copyright (c) 2017 Virtue Studios
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { EventType } from 'engine/enums/event-type';
+import _events from 'engine/events';
+
+import { sendMessage } from 'shared/chat';
+import { chatplayer, mesbox } from 'shared/dialog';
+
+import { hasFinished } from '../quest';
+
+
+
+	    _events.bindEventListener(EventType.OPLOC1, 2371, (ctx) => {
+	        sendMessage(ctx.player, "Digsite infomation and exam centre.");
+	    });
+
+		_events.bindEventListener(EventType.OPLOC1, 4066, (ctx) => {
+	        mesbox(ctx.player, "You inspect the signpost more closely. You see nothing wrong with it<br> initially, but after some time you notice teeth marks in the wood. It looks<br> like people have tried eating the post.");
+	    });
+
+		_events.bindEventListener(EventType.OPLOC1, 5164, (ctx) => {
+	        mesbox(ctx.player, "The sinepost has a note pinned onto it. The note says:<br> '----Braindead Butler Wanted----<br> Gravedigging skills essential - Hunchback advantageous<br> See Dr Fenkenstrain at the castle NE of Canifis'");
+	    });
+
+		_events.bindEventListener(EventType.OPLOC1, 25397, (ctx) => {
+	        mesbox(ctx.player, "~ The Observatory ~<br> Step ahead to the reception if you wish to explore RuneScape's most<br> magnificent invention.")
+			chatplayer(ctx.player, "Magnificent invention? I've seen some pretty magnificent<br> things in my time. It'll have to be pretty impressive.")
```

Comment:
You'll need to use async/await here:
```
await (ctx) => {
    await mesbox(ctx.player, "~ The Observatory ~<br> Step ahead to the reception if you wish to explore RuneScape's most<br> magnificent invention.");
    await chatplayer(ctx.player, "Magnificent invention? I've seen some pretty magnificent<br> things in my time. It'll have to be pretty impressive.");
}
```

Basically, you have to use it anywhere you have `mesbox`, `chatplayer`, `chatnpc`, etc and something else after it (otherwise anything after will never process). Worth double-checking the signpost to make sure both messages definitely display